docs/tutorials/enable-totp-openstack.md

```diff
@@ -0,0 +1,93 @@
+# Configuring Time-Based One-Time Password (TOTP) on OpenStack
+
+Author: Ramon Grullon
+
+## Introduction
+
+Horizon added time-based one-time password (TOTP) authentication support,
+leveraging the already existing two factor authentication from Keystone.
+Now, if a user activates TOTP on Keystone, it gets activated on Horizon too.
+
+## Prerequesites
+
+- A working OpenStack 2023.2 environment deployed using kolla-ansible.
+- Enabled totp on all control nodes
```

Comment:
Should we explain how to run kolla-ansible against a cloud to enable TOTP in keystone? Maybe a mention of hey you need make this configuration change to keystone then run kolla-ansible with `--tags keystone` seems relevant to me.

Reply:
fixed